snaps/reference/permissions.md

```diff
@@ -107,13 +107,17 @@ Specify this permission in the manifest file as follows:
 
 ### `endowment:keyring`
 
-For a dapp to call [Keyring API](../features/custom-evm-accounts/index.md) methods on an account management Snap,
+For a dapp to call [Account Management API](keyring-api/account-management/index.md) methods on an
+account management Snap to integrate [custom EVM accounts](../features/custom-evm-accounts/index.md),
 the Snap must configure a list of allowed dapp URLs using the `endowment:keyring` permission.
+This permission grants a Snap access to Account Management API requests sent to the Snap, using the
```

Comment:
Maybe this should come before the previous paragraph since it explains what this permission is. Feel free to ignore this suggestion if you don't agree with it!